src/app/GitExtUtils/GitUI/Theming/ThemeSettings.cs

```diff
@@ -26,5 +26,8 @@ public ThemeSettings(
         public string[] Variations { get; }
 
         public bool UseSystemVisualStyle { get; }
+
+        public bool IsDarkTheme =>
+            Theme.GetNonEmptyColor(KnownColor.Window).GetBrightness() < 0.5;
```

Comment:
There's a dedicated API in Windows Forms for that, isn't there?

Reply:
This is to know if the theme is dark, this is what was there.
I considered adding a specific color just to enable or a special setting to enable. We want to enable dark mode separate from the system for some time, there will be many visuals.

If we go with .net9 this can be merged if the most severe issues are fixed as experimental, there may be designers helping...
But also that will take a long time to get to.